scripts/azureml-assets/azureml/assets/environment/build.py

```diff
@@ -62,6 +64,16 @@ def create_acr_task(image_name: str,
             'id': 'test',
             'cmd': f"$Registry/{image_name} {test_command}"
         })
+        task['steps'].append({

```

Comment:
If `trivy_url` is optional, you should check for `trivy_url is not None` before appending a scanning step. Maybe just use the URL to switch scanning on or off? If it's specified, then do vuln scanning. If not, don't?